docs/md-enhance/src/zh/guide/playground.md

```diff
@@ -133,6 +141,17 @@ Vue 预设默认使用官方 playground，并不像 [Vue Playground](./vue-playg
 
 :::
 
+::: info UnoCSS 预设
+
+UnoCSS 预设默认使用官方 playground，通过 `@file` 的不同文件类型，对应官方 playground 的相应输入，每个文件类型仅支持一个文件，具体：
```

Comment:
It's not clear for the representation "每个文件类型仅支持一个文件"

Reply:
It's better to force them to `index.html` `config.js` and `style.css` I guess

Reply:
I have changed it in the doc. Suggest to set `index.html` `config.js` `style.css`. But not make force seting in code, Just in case someone has spell error, and the demo is also working.